src/network/networkd-network.h

```diff
@@ -164,6 +164,7 @@ struct Network {
         bool allow_port_to_be_root;
         bool unicast_flood;
         unsigned cost;
+        unsigned priority;
```

Comment:
(and yeah, "cost" appears broken too, the netlink type appears to be uint32_t, and not unsigned, and we should use uint32_t here hence too...)